storm-webapp/src/main/java/org/apache/storm/daemon/ui/WEB-INF/js/script.js

```diff
@@ -332,6 +332,14 @@ var makeWorkerStatsTable = function (response, elId, parentId, type) {
                     data;
             }
         },
+        {
+            data: 'owner',
```

Comment:
Will this also add "owner" column to the topology summary page? If so, it will be redundant since the topology summary page already has "owner" information

Reply:
Correct. This also adds owner to the topology summary page. The duplicate owner can be removed at the cost of adding logic to script.js. It seems a wash to me.

Reply:
Removing duplicate owner is simple. Making this change and also removing "owner" tooltip from  topology-page-template